actions/utils.go

```diff
@@ -5,6 +5,19 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
+	"math"
+	"math/rand"
```

Comment:
This is only pseudo random, we should seed it. Probably with a hashed timestamp:

https://golang.org/pkg/hash/
```go
rand.Seed(hash(time.Now().UTC().UnixNano()))
```

Reply:
I am not sure but I believe that seeding could be added in main.go, and would be applicable through the application.

Reply:
Agree, let me look into that. Mind filing an issue for that?

Reply:
https://github.com/oysterprotocol/brokernode/issues/217